aquatic_udp/src/workers/socket/storage.rs

```diff
@@ -118,104 +118,109 @@ impl PendingScrapeResponseSlab {
     }
```

Comment:
This is a pretty significant change, is there a way these changes can be made in a different file?

Reply:
I managed to make the diff smaller here, but the effect is the same: harness the quickcheck test code as if it were a fuzz harness. I've found that quickcheck only runs a couple dozen iterations per run, and sometimes fuzzing it for longer turns up results.